src/content/docs/getting-started/start-from-scratch.md

```diff
@@ -8,51 +8,85 @@ title: Start From Scratch
 
 Register at [app.testomat.io](https://app.testomat.io) and activate your user account. Then create a new project.
 
-![Create New Project](./images/scratch1.png)
+![Create New Project](./images/New_qIJOdkr6_2024-07-09.png)
 
 The type of your new project will influence the look of your interface, as well as its intended audience. Choose **BDD Project** if you plan to follow BDD descriptions or use the Cucumber framework in future. If you haven't decided yet, you can start with default **Classical Project**. Depending on your choice, the interface will differ, as follows:
 
 * Within a **BDD Project**, you will have feature definitions written both in the source code (by engineers) and in Testomatio (by managers and QAs). So you will be able to track and plan Cucumber automation, as well as synchronize the actual feature files with relevant test cases. When a scenario has changed in the system, but has not been updated in the code yet, you will be notified that this certain scenario is out of date.
 * Within a **Classical Project**, test cases will be written in free form using markdown. Automated tests will be synced up with test cases, so you will be able to see the test description in one tab and the corresponding test code in another tab. When a description of a test case undergoes changes, you will be notified that a certain test might need to be updated.
 
-![Select type of project](./images/scratch2.png)
+Let's start creating a project:
 
-Fill the name of your project in the **Project Title** field.  The **Repository URL** field is optional, so you can simply skip it. Click the **Create** button to start a new project. 
+1. Fill the name of your project in the **Project Title** field.  
+2. The **Fill demo data** checkbox is optional. This option fills the project with ready-made test data so that new users can try testomat.io quickly and easily. You can simply skip it. 
+3. Click the **Create** button to start a new project. 
 
-![New blank project](./images/scratch3.png)
+![Select type of project](./images/New_rughOXfd_2024-07-09.png)
+
+Congratulations! You have just created your first project. Now you can start creating suites and test cases for your projects.
+
+![New blank project](./images/New_WG8caz4S_2024-07-09.png)
 
 ## Write First Test Cases
 
-Now you can start creating suites and test cases for your projects. We have implemented Bulk Creation input, which enables you to create as many test suites as you need:
+Let's take a look at creating test cases step by step.
+
+1. Fill the name in the input field. 
+2. Click the **+ Suite** button to create a test case. 
+
+![Create test suite](./images/New_0g8uRrQa_2024-07-10.png)
+
+We have implemented **Bulk edit**, which enables you to create as many test suites as you need:
```

Comment:
Let's not use `We` and stay more formal here

As it's not clear who are "we":

* reader + author
* developers 

Also, I think Bulk Edit is an advanced topis and should be removed at all from Getting Started